crux_macros/src/bridge.rs

```diff
@@ -0,0 +1,107 @@
+use proc_macro2::TokenStream;
+use quote::quote;
+use syn::ItemStruct;
+
+pub fn bridge_impl(input: ItemStruct) -> TokenStream {
+    let ident = &input.ident;
+    quote! {
+        #[derive(Default)]
```

Comment:
I don't think the derive is guaranteed to work for all `Apps` - some may have fields with non-derivable `Default` and will need to hand implement `Default`, we probably should leave writing this line up to the user